.travis.yml

```diff
@@ -2,6 +2,9 @@ language: python
 
 before_script:
 - pip install tox
+- sed -i 's/\/opt/\/home\/travis\/build\/rcbops/g' ansible-role-requirements.yml
+- pip2 install --force-reinstall 'ansible===1.9.4'
+- ansible-galaxy install --role-file=ansible-role-requirements.yml --ignore-errors --force
```

Comment:
I will say it wasn't removed when jesse updated the line above it in https://github.com/openstack/openstack-ansible/commit/8bf181ebf184ba6e8d3efcd3e191cdcdd17a8c12